### PR TITLE
Added additional docstrings to undocumented functions

### PR DESCRIPTION
**Pull request by fluencydoc**
# Description

Added ~30 docstrings to undocumented functions throughout the repo.

## Type of change

- Other

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and test it
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

## Screenshots (if any)
N/A

### CODE REVIEW

File: extra/update-version.js [hunk 1]
```diff
@@ -37,6 +37,12 @@ if (! exists) {
     console.log("version exists");
 }
 
+/**
+ * Updates the version number in package.json and commits it to git
```
Comment (Saibamen) [inline extra/update-version.js:41]: Missing dot at the end

File: server/modules/apicache/apicache.js [hunk 2]
```diff
@@ -68,6 +68,15 @@ function ApiCache() {
...
     this.id = instances.length;
 
+    /**
+     * Logs a message to the console if the `DEBUG` environment variable is set.
+     * @param {string} a - The first argument to log.
+     * @param {string} b - The second argument to log.
+     * @param {string} c - The third argument to log.
+     * @param {string} d - The fourth argument to log, and so on... (optional)
+     *
+     * Generated by Trelent
```
Comment (Saibamen) [inline server/modules/apicache/apicache.js:78]: Do we need this in all jsdoc comments?

Comment (fluencydoc) [reply server/modules/apicache/apicache.js:78]: We appreciate recognition for writing the js doc comments, but if you'd like we can make it a bit more subtle! We are open to suggestions in this regard.

Comment (Saibamen) [reply server/modules/apicache/apicache.js:78]: For me, I can live with that

Comment (fluencydoc) [reply server/modules/apicache/apicache.js:78]: Great, it is much appreciated!

**Comment by fluencydoc** [comment]
@Saibamen  Are we good to go ahead with the workflow approval to run required checks?

**Comment by Saibamen** [comment]
Which required checks?

**Comment by fluencydoc** [comment]
I need a maintainer to [approve running the Actions workflow](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks) as I am a first-time contributor.

**Comment by Saibamen** [comment]
I don't understand. Everything is fine. All tests are passed

image

**Review by Saibamen — APPROVED** [review]

**Comment by fluencydoc** [comment]
Okay, looks like we're good to go then! Thanks @Saibamen

**Pull request merged**